### ISSUE TITLE
[Item][Spell] Proc rate/On use effects of pre-TBC items.

### ISSUE THREAD

**Issue by tsu-1** [Atlantiss/NetherwingBugtracker#3062]
This is reserved space for report of items proc rate and on use effects for pre-TBC items. (MC, BWL, Onyxia, ZG, AQ20, AQ40, Naxx, BoE Items 30-60.

Addon for PPM testing: https://github.com/ZsoltMolnarrr/Procodile-TBC/releases (by ZsoltMolnarr)

- [ ] Comment 1
- [ ] Comment 2
- [ ] Comment 3

**Comment by huntdfl** [comment]
(Badge of the Swarmguard)[https://tbc-twinhead.twinstar.cz/?item=21670]
All procs are counted from the 30 second duration of trinket
Test 1: (Auto-attacks only) 1 proc(s) in 30 seconds
Test 2: (Auto-attacks only) 1 proc(s) in 30 seconds
Test 3: (Auto-attacks w/ Hamstring spam) 4 proc(s) in 30 seconds
Test 4: (Auto-attacks w/ Hamstring spam + Windfury totem) 6 proc(s) in 30 seconds
Test 5: (Auto-attacks w/ Hamstring + Windfury) 4 proc(s) in 30 seconds
Test 6: (Auto-attacks w/ Hamstring + Windfury) 8 proc(s) in 30 seconds - Max (6) stacks within first 15s
Test 7: (Auto-attacks w/ Hamstring + Windfury) 6 proc(s) in 30 seconds
Test 8: (Auto-attacks w/ Dual Wield + Windfury) 4 proc(s) in 30 seconds
Test 8: (Auto-attacks w/ Dual Wield + Windfury) 1 proc(s) in 30 seconds

Thoughts: Proc rate seems low and chance to trigger on autos seems extremely off.

**Comment by huntdfl** [comment]
(Bonereaver's Edge)[https://tbc-twinhead.twinstar.cz/?item=17076]
Proc ID# 21153 - Proc can grant (3) charges ignoring 700/1400/2100 armor

Test 1:
Duration: 10 minutes - Auto attacks only
Total procs: 7 (1x debuff only - never reached 3 stacks)
PPM: 0.69

Test 2:
Duration: 6 minutes - Hamstring spam + Windfury Totem
Total Procs: 7 - (X represents 1-2-3 buff) 3x, 2x, 1x, 1x, 3x, 3x, 3x
PPM: 1.52

**Comment by Thugli** [comment]
Alcor's Sunrazer

Item Name/y | Current PPM | Blizzlike PPM | Does it Scale with SP/AP?
-- | -- | -- | --
Alcor's Sunrazor | 2.7 |   | Yes, tooltip says 75-105, my average non-crit was 552. I have 496 bonus shadow damage.

 
![image](https://user-images.githubusercontent.com/37326256/49485302-6f90fd80-f808-11e8-94ee-49488eb25e7a.png)

**Comment by Thugli** [comment]
Alcor's Sunrazor | 2.7 |   | Yes, tooltip says 75-105, my average non-crit was 552. I have 496 bonus shadow damage. |   |   |   |  
-- | -- | -- | -- | -- | -- | -- | --
Anathema/Benediction |   |   |   |   |   |   |  
Ancient Cornerstone Grimoire |   |   |   |   |   |   | Working as intended, summoned skeleton's damage needs to be researched and compared.-Check skeleton's average melee damage and successful melee attack %. Skeleton seemed to miss relatively often.
Emerald Dragonfang | 1 |   | Yes, tooltip says 87-105, my average was 308. I have 496 bonus nature damage. |   |   |   | Every proc hit and none of them crit. Test Again for a longer period of time to look in to this. Should this proc be able to crit? Should it be able to miss/resist?
Empyrean Demolisher | 0.76 |   |   |   |   |   | Does the haste buff stack properly?
Eye of Dimunition |   |   |   |   |   |   |  
Eye of Moam |   |   |   |   |   |   |  
Fetish of Chitinous Spikes |   |   | No, Tooltip says 25 nature damage per tick & itdoes 25 damage per tick. I have +660 bonus nature damage. |   |   |   | Working as intended; ticks going off after every attack for 25 nature damage. Blizzlike to not scale with SP.
Fetish of the Sand Reaver |   |   |   |   |   |   |  
Freezing Band |   |   |   |   |   |   | Working as intended, proc chance needs for testing
Gut ripper | 1.3 |   | Unsure, tooltip says 95-121, my average non-critwas 63. I have 34 melee attack power. |   |   |   |  
Hand of Edward the Odd | 1.9 |   |   |   |   |   | The proc gives 320 haste rating, which equates to 20% increased haste. Rank 2 greater heal is normally a 3 second cast, with the proc, it became 2.5 seconds. 20% of 3 =0.6, meaning the cast time should have been reduced to 2.4 seconds not 2.5. When I tested with mind blast, which normally has a cast time of 1.5 seconds, it properly reduced the cast time to 1.2 which is 20% faster as it should be. I will have to test on more abilities and consider the possibility that I encoutered a visual bug.
Lei of Lillies |   |   |   |   |   |   | Working as intended; health & mana gain are within blizzlike range.
Lifestone |   |   |   |   |   |   | On-use effect functions properly and heals within the listed range.
Mar'li's Eye |   |   |   |   |   |   | Working as intended; gaining 60 mp5 for 30 seconds, for a total of 6 procs.
Robes of Insight |   |   |   |   |   |   | Working as intended; reduces mana cost of next spell cast by up to 500 mana
Scarab Brooch |   |   |   |   |   |   | Working as intended; persistent shield' effect creates a shield that absorbs 15% of the previous heal casted on the target.
Shadowblade | 1.7 |   | Yes, tooltip says 110-140, my average non-crit was 552. I have 675 bonus shadow damage. |   |   |   |  
Talisman of Ascendance |   |   |   |   |   |   | Talisman of Ascendance on use effect is only triggered by direct damage or healing spell casts, despite the tooltip saying "Your next 5 damaging or healing spells cast". Needs research. Bonus healing stacks properly on direct heals.
Talisman of Ephemeral Power |   |   |   |   |   |   |  
The Lion Horn of Stormwind |   |   |   |   |   |   |  
The Night Blade | 1.6 |   |   |   |   |   |  
Vanquished Tentacle of C'thun |   |   |   |   |   |   | Missed every attack on non-elite target dummy & every mob I have tested it on. Unlike Ancient cornerstone grimoire (Both summon 'guardian' npcs), this npc enganges in combat  with a neutral mobs if I attack it. Whereas the anicient cornerstoen grimoire guardian will only attack a hostile/red mob that is attacking me. Will need research on what the proper combat rules are for guardian type minions are.
Zandalarian Hero Charm |   |   |   |   |   |   | Working as intended; stacks only drop from direct damage spells.

**Comment by Thugli** [comment]
Item Name | Current PPM | Blizzlike PPM | Does it Scale with SP/AP? | Additional Notes
-- | -- | -- | -- | --
Alcor's Sunrazor | 2.7 |   | Yes, tooltip says 75-105, my average non-crit was 552. I have 496 bonus shadow damage. |  
Anathema/Benediction |   |   |   |  
Ancient Cornerstone Grimoire |   |   |   | Working as intended, summoned skeleton's damage needs to be researched and compared.-Check skeleton's average melee damage and successful melee attack %. Skeleton seemed to miss relatively often.
Emerald Dragonfang | 1 |   | Yes, tooltip says 87-105, my average was 308. I have 496 bonus nature damage. | Every proc hit and none of them crit. Test Again for a longer period of time to look in to this. Should this proc be able to crit? Should it be able to miss/resist?
Empyrean Demolisher | 0.76 |   |   | Does the haste buff stack properly?
Eye of Dimunition |   |   |   |  
Eye of Moam |   |   |   |  
Fetish of Chitinous Spikes |   |   | No, Tooltip says 25 nature damage per tick & itdoes 25 damage per tick. I have +660 bonus nature damage. | Working as intended; ticks going off after every attack for 25 nature damage. Blizzlike to not scale with SP.
Fetish of the Sand Reaver |   |   |   |  
Freezing Band |   |   |   | Working as intended, proc chance needs for testing
Gut ripper | 1.3 |   | Unsure, tooltip says 95-121, my average non-critwas 63. I have 34 melee attack power. |  
Hand of Edward the Odd | 1.9 |   |   | The proc gives 320 haste rating, which equates to 20% increased haste. Rank 2 greater heal is normally a 3 second cast, with the proc, it became 2.5 seconds. 20% of 3 =0.6, meaning the cast time should have been reduced to 2.4 seconds not 2.5. When I tested with mind blast, which normally has a cast time of 1.5 seconds, it properly reduced the cast time to 1.2 which is 20% faster as it should be. I will have to test on more abilities and consider the possibility that I encoutered a visual bug.
Lei of Lillies |   |   |   | Working as intended; health & mana gain are within blizzlike range.
Lifestone |   |   |   | On-use effect functions properly and heals within the listed range.
Mar'li's Eye |   |   |   | Working as intended; gaining 60 mp5 for 30 seconds, for a total of 6 procs.
Robes of Insight |   |   |   | Working as intended; reduces mana cost of next spell cast by up to 500 mana
Scarab Brooch |   |   |   | Working as intended; persistent shield' effect creates a shield that absorbs 15% of the previous heal casted on the target.
Shadowblade | 1.7 |   | Yes, tooltip says 110-140, my average non-crit was 552. I have 675 bonus shadow damage. |  
Talisman of Ascendance |   |   |   | Talisman of Ascendance on use effect is only triggered by direct damage or healing spell casts, despite the tooltip saying "Your next 5 damaging or healing spells cast". Needs research. Bonus healing stacks properly on direct heals.
Talisman of Ephemeral Power |   |   |   |  
The Lion Horn of Stormwind |   |   |   |  
The Night Blade | 1.6 |   |   |  
Vanquished Tentacle of C'thun |   |   |   | Missed every attack on non-elite target dummy & every mob I have tested it on. Unlike Ancient cornerstone grimoire (Both summon 'guardian' npcs), this npc enganges in combat  with a neutral mobs if I attack it. Whereas the anicient cornerstoen grimoire guardian will only attack a hostile/red mob that is attacking me. Will need research on what the proper combat rules are for guardian type minions are.
Zandalarian Hero Charm |   |   |   | Working as intended; stacks only drop from direct damage spells.

**Comment by knasen88** [comment]
I saw a guy using darkmantle dungeon 2 set with what seemed like 200ppm
if unfixed i can easily see it being overhwelmingly strong.

**Comment by Evolvee** [comment]
@Tereneckla @Manaci tldr pls 🤓

**Comment by Tereneckla** [comment]
tldr: https://tbc-twinhead.twinstar.cz/?item=19949 should have no icd on charge removal
https://db.rising-gods.de/?item=21579 misses with every attack
https://classic.wowhead.com/item=22678/talisman-of-ascendance should advance stacks with dots/hots

Then there are some items that need to tested in a controlled environment.
And a table of PPM tests of various items with no further comments.
